src/gui/Lv2ViewBase.cpp

```diff
@@ -51,13 +52,51 @@ namespace lmms::gui
 {
 
 
-Lv2ViewProc::Lv2ViewProc(QWidget* parent, Lv2Proc* ctrlBase, int colNum) :
-	LinkedModelGroupView (parent, ctrlBase, colNum)
+static int numDigitsAsInt(float f)
 {
-	class SetupWidget : public Lv2Ports::ConstVisitor
+	// use rounding:
+	// LcdSpinBox sometimes uses roundf(), sometimes cast rounding
+	// we use rounding to be on the "safe side"
+	const float rounded = roundf(f);
+	int asInt = static_cast<int>(rounded);
+	int digits = 1; // always at least 1
+	if(asInt < 0)
+	{
+		++digits;
+		asInt = -asInt;
+	}
+	// "asInt" is positive from now
+	int power = 10;
+	for(int i = 1; i<6; ++i, power *= 10)
```

Comment:
Why only 6?

Reply:
What do you suggest? Using 10 and `int32_t`?

Reply:
That sounds better for potential future usages.